### PR TITLE
Add quarkus qe beefy test suite

### PR DESCRIPTION
**Pull request by Sgitario**

### CODE REVIEW

**Comment by Sgitario** [comment]
cc @rsvoboda

**Review by geoand — APPROVED** [review]
LGTM.

We should be able to see the results of the run tomorrow

File: quarkus-qe-beefy/info.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+url: https://github.com/quarkus-qe/beefy-scenarios
+issues:
+  repo: quarkusio/quarkus
+  latestCommit: 15409
```
Comment (gsmet) [inline quarkus-qe-beefy/info.yaml:4]: Hmmm, no, that's not correct. This needs to point to an issue that will be handled by CI.

Something similar to this one: https://github.com/quarkusio/quarkus/issues/11515 .

Comment (geoand) [reply quarkus-qe-beefy/info.yaml:4]: @Sgitario please follow up with a new PR when you have the proper issue created

Comment (Sgitario) [reply quarkus-qe-beefy/info.yaml:4]: Sorry, my bad. I thought the issue number was just like a markdown. It's corrected now in: https://github.com/quarkusio/quarkus-ecosystem-ci/pull/43

**Pull request merged**